README.md

```diff
@@ -47,16 +47,62 @@ Full set up is in https://github.com/nodejs/build/tree/master/setup/www minus se
 
 All of the Node.js Foundation websites, including this repo, are jointly governed by the **Website Working Group**. See [GOVERNANCE.md](./GOVERNANCE.md) to learn more about the group's structure and [CONTRIBUTING.md](./CONTRIBUTING.md) for guidance about the expectations for all contributors to this project.
 
-### Website Working Group Members
+### Website Working Group Admins
+
+- Frederic Hemberger ([fhemberger](https://github.com/fhemberger)): [@fhemberger](https://twitter.com/fhemberger), `mail``@``frederic-hemberger.de`
+- Mikeal Rogers ([mikeal](https://github.com/mikeal)): [@mikeal](https://twitter.com/mikeal), `mikeal.rogers``@``gmail.com`
```

Comment:
Ahhh... As all of our emails are public via the git log, we can't really hide :) johphi@gmail.com and rod@vagg.org.